.bumpversion.cfg

```diff
@@ -0,0 +1,35 @@
+[bumpversion]
+current_version = 0.21.0.dev0
+files = src/py/pyodide/__init__.py src/py/setup.cfg pyodide-build/setup.cfg docs/conf.py
```

Comment:
bump2version rewrites the config file, but the pre-commit doesn't like its formatting. So I moved it to a separate config file and make pre-commit to ignore it.

Anyway, I am thinking of writing a short script instead of using `bump2version`, because `bump2version` doesn't seem to work intuitively to me when dealing with prerelease of dev versions.